h2o-py/h2o/explanation/_explain.py

```diff
@@ -1001,7 +1068,7 @@ def _handle_ice(model, frame, colormap, plt, target, is_factor, column, show_log
                 cols=[column],
                 plot=False,
                 targets=target,
-                nbins=100 if not is_factor else 1 + frame[column].nlevels()[0]
+                nbins=nbins if not is_factor else 1 + frame[column].nlevels()[0]
```

Comment:
nit: Consider the usage of parenthesis to separate the complicated ternary. Even though operator precedence rules are clear, it's easier to visually parse sth like 
```python
nbins=nbins if not is_factor else (1 + frame[column].nlevels()[0])
```

Reply:
[f2bd46e](https://github.com/h2oai/h2o-3/pull/6119/commits/f2bd46e90cf03fc39da9cffae0cbe66a7064279c)